These are new in Soprano 2.7.

--- origsrc/pykde4-4.7.0/sip/soprano/nao.sip	2011-06-22 10:02:34.000000000 -0500
+++ src/pykde4-4.7.0/sip/soprano/nao.sip	2011-08-07 06:04:28.898203100 -0500
@@ -69,8 +69,6 @@ QUrl                    version ();
 QUrl                    deprecated ();
 QUrl                    isDataGraphFor ();
 QUrl                    userVisible ();
-QUrl                    Agent ();
-QUrl                    maintainedBy ();
 };
 // NAO
 
